examples/with-next-auth/pages/api/auth/[...nextauth].js

```diff
@@ -0,0 +1,105 @@
+import NextAuth from 'next-auth'
+import Providers from 'next-auth/providers'
+
+const options = {
+  site: process.env.VERCEL_URL || 'http://localhost:3000',
```

Comment:
A fallback to `http://localhost:3000` shouldn't be required, if `process.env.VERCEL_URL` is not provided it should fallback to undefined and fail the deployment.